src/subdomains/core/payment-link/services/payment-activation.service.ts

```diff
@@ -148,7 +150,7 @@ export class PaymentActivationService implements OnModuleInit {
     const { paymentRequest, paymentHash } =
       transferInfo.method === Blockchain.LIGHTNING
         ? await this.createLightningRequest(payment, transferInfo, expirySec)
-        : await this.createEvmRequest(transferInfo);
+        : await this.createBlockchainRequest(transferInfo);

```

Comment:
Move `this.createLightningRequest(payment, transferInfo, expirySec)` into method `createBlockchainRequest`?